deps/build.jl

```diff
@@ -0,0 +1,4 @@
+default_dir = get(ENV, "CONDA_JL_HOME", abspath(dirname(@__FILE__), "usr"))
```

Comment:
This would enable the use of an existing conda environment, so that you can use an environment like `julia` in your system-wide environment.